### DISCUSSION TITLE
delirium assessment method  used for the ITEMID (228332)

### DISCUSSION THREAD

**Discussion by scorpioneuro** [MIT-LCP/mimic-code#1711] [MIMIC-IV] [answered]
Hi Support Team,
In MIMIC-IV,  there is an ITEMID 228332 that says "delirium assessment". 
Can you tell me what assessment method was used for the ITEMID (228332)? 
Is it CAM-ICU?

**Comment by alistairewj** [comment]
Are there any rows for that itemid? If I remember there was an empty itemid for delirium assessment.

Either way yes they do use CAM-ICU. There are at least two itemid (at least, if I recall some components may be stored separately as yes/no answers). Mid way through data collection they removed the ability to document CAM-ICU as "Unable to assess", i.e. there had to be a yes/no decision. This increased the rate of delirium. Just something to be aware of.

**Reply by alistairewj** [reply, marked as answer]
Also see #1652

**Reply by scorpioneuro** [reply]
Thanks for your kind reply.

**Reply by aircondition426** [reply]
I also have the same issue with you. Each item under the RASS scale has information collected at multiple time points, and there is no detailed scoring record in it. There is only Yes or No label. Does Yes mean positive for the event and No mean negative for the event? There are multiple time points in one item of the same icu-stay, including Yes or No. Is there an event that is judged as positive in only one time point? The incidence of delirium in that way is very high, even to more than 90%, which does not seem to be consistent with the highest incidence of delirium mentioned in the literature I read as high as 20%.
